### DISCUSSION TITLE
VEP cache version update

### DISCUSSION THREAD

**Discussion by proteinosome** [broadinstitute/seqr#2500] [Q&A]
Hi, thank you for the development of this software package. I'm currently testing this out and have a quick question: Is it possible to update the VEP cache data to a newer version? I tried directly downloading v105 data into the seqr data dir but the pipeline threw an error saying that it expects version 99 in the annotation step.

Thank you.

**Comment by hanars** [comment]
Can you provide the stacktrace for the error you are seeing in the annotation step?

**Comment by proteinosome** [comment]
Hi I managed to upgrade it by going into the docker container and updating VEP to version 105. Basically, version 105 of the database requires VEP version 105. The VEP in the docker container is version 99.

Thank you!
